compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java

```diff
@@ -2091,6 +2095,25 @@ void createOnClause(DiagnosticPos pos, Set<Whitespace> ws) {
         joinClause.onClause = onClause;
     }
 
+    void createOrderByKey(DiagnosticPos pos, Set<Whitespace> ws, boolean isAscending) {
+        BLangOrderKey orderKey = (BLangOrderKey) TreeBuilder.createOrderKeyNode();
+        orderKey.pos = pos;
+        orderKey.setOrderKey(this.exprNodeStack.pop());
+        orderKey.setOrderDirection(isAscending);
+        orderKeyListStack.push(orderKey);
+    }
+
+    void createOrderByClause(DiagnosticPos pos, Set<Whitespace> ws) {
+        BLangOrderByClause orderByClause = (BLangOrderByClause) TreeBuilder.createOrderByClauseNode();
+        orderByClause.addWS(ws);
+        orderByClause.pos = pos;
+        Collections.reverse(orderKeyListStack);
+        while (orderKeyListStack.size() > 0) {
```

Comment:
try to use .isEmpty() rather when ever possible. these provided functions are more optimized

Reply:
`BLangPackageBuilder.java` is no longer used